### PR TITLE
added JS file with math operations

### PR DESCRIPTION
**Pull request by olyatmchshn**

### CODE REVIEW

File: homeworks/olha.tymchyshyn_olyatmchshn/4-basics/basics.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+console.log(`Addition: ${a} + ${b} = ${add}`);
+
+const diff = b - c;
+console.log(`Subtraction: ${b} - ${c} = ${diff}`);
+
+const multiply = y * z;
+console.log(`Multiplication: ${y} * ${z} = ${multiply}`);
+
+const div = c / z;
+console.log(`Division: ${c} / ${z} = ${+div.toFixed(2)}`);
```
Comment (vladyslav-yermolin-moc) [inline homeworks/olha.tymchyshyn_olyatmchshn/4-basics/basics.js:18]:
```suggestion
console.log(`Division: ${c} / ${z} = ${div.toFixed(2)}`);
```
Hey, @olyatmchshn, there is no necessity to make a number out of `div.toFixed(2)` by using `+` in this case.

File: homeworks/olha.tymchyshyn_olyatmchshn/4-basics/basics.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+const expo = c ** 2;
+console.log(`Exponentiation: ${c} ** ${2} = ${expo}`);
```
Comment (vladyslav-yermolin-moc) [inline homeworks/olha.tymchyshyn_olyatmchshn/4-basics/basics.js:21]: There should be some variable from your list in place of `2`

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

File: homeworks/olha.tymchyshyn_olyatmchshn/4-basics/basics.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+const a = 24 + 68;
+const b = 37 - 16;
+const c = 100 / 5;
+const x = 4 * 16;
+const y = 4 ** 2;
+const z = 23 % 5;
+
+const add = a + b;
```
Comment (vladyslav-yermolin-moc) [inline homeworks/olha.tymchyshyn_olyatmchshn/4-basics/basics.js:8]: The common practice is to make a variable, if you like to use it twice at least. 
So, all these vars `add`, `diff`, etc are redundant.
You can just calculate them on the fly:
```js
console.log(`Addition: ${a} + ${b} = ${a + b}`);
```

File: homeworks/olha.tymchyshyn_olyatmchshn/4-basics/basics.js (hunk 1 above)
Comment (vladyslav-yermolin-moc) [reply homeworks/olha.tymchyshyn_olyatmchshn/4-basics/basics.js:18]: Anyway it became a sting as a part of `Division: ${c} / ${z} = ${div.toFixed(2)}`

**Review by electrovladyslav — APPROVED** [review]

**Review by vladyslav-yermolin-moc — APPROVED** [review]

**Pull request merged**